### ISSUE TITLE
[STTools] python names ending with an underscore confuse sphinx (Trac #1738)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1730]
A number of pybinding objects have names which end with an underscore. Ending a name with an underscore (when not starting it with an underscore) is uncommon in python and confuses sphinx.

```text
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:10: ERROR: Unknown target name: "seed_with_all_core_hits_i3domlaunch".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:12: ERROR: Unknown target name: "seed_with_hlc_core_hits_i3domlaunch".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:14: ERROR: Unknown target name: "seed_with_omkey_hits_i3domlaunch".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:16: ERROR: Unknown target name: "seed_with_nth_omkey_hits_i3domlaunch".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:18: ERROR: Unknown target name: "seed_with_hit_series_map_hits_from_frame_i3domlaunch".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:10: ERROR: Unknown target name: "seed_with_all_core_hits_i3recopulse".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:12: ERROR: Unknown target name: "seed_with_hlc_core_hits_i3recopulse".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:14: ERROR: Unknown target name: "seed_with_omkey_hits_i3recopulse".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:16: ERROR: Unknown target name: "seed_with_nth_omkey_hits_i3recopulse".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:18: ERROR: Unknown target name: "seed_with_hit_series_map_hits_from_frame_i3recopulse".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:10: ERROR: Unknown target name: "seed_with_all_core_hits_i3recopulse".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:12: ERROR: Unknown target name: "seed_with_hlc_core_hits_i3recopulse".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:14: ERROR: Unknown target name: "seed_with_omkey_hits_i3recopulse".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:16: ERROR: Unknown target name: "seed_with_nth_omkey_hits_i3recopulse".
/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:18: ERROR: Unknown target name: "seed_with_hit_series_map_hits_from_frame_i3recopulse".
```

Migrated from https://code.icecube.wisc.edu/ticket/1738
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:38", 
    "description": "A number of pybinding objects have names which end with an underscore. Ending a name with an underscore (when not starting it with an underscore) is uncommon in python and confuses sphinx.\n{{{\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:10: ERROR: Unknown target name: \"seed_with_all_core_hits_i3domlaunch\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:12: ERROR: Unknown target name: \"seed_with_hlc_core_hits_i3domlaunch\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:14: ERROR: Unknown target name: \"seed_with_omkey_hits_i3domlaunch\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:16: ERROR: Unknown target name: \"seed_with_nth_omkey_hits_i3domlaunch\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_DOMLaunch_:18: ERROR: Unknown target name: \"seed_with_hit_series_map_hits_from_frame_i3domlaunch\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:10: ERROR: Unknown target name: \"seed_with_all_core_hits_i3recopulse\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:12: ERROR: Unknown target name: \"seed_with_hlc_core_hits_i3recopulse\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:14: ERROR: Unknown target name: \"seed_with_omkey_hits_i3recopulse\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:16: ERROR: Unknown target name: \"seed_with_nth_omkey_hits_i3recopulse\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulseMask_:18: ERROR: Unknown target name: \"seed_with_hit_series_map_hits_from_frame_i3recopulse\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:10: ERROR: Unknown target name: \"seed_with_all_core_hits_i3recopulse\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:12: ERROR: Unknown target name: \"seed_with_hlc_core_hits_i3recopulse\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:14: ERROR: Unknown target name: \"seed_with_omkey_hits_i3recopulse\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:16: ERROR: Unknown target name: \"seed_with_nth_omkey_hits_i3recopulse\".\n/Users/kmeagher/icecube/combo/release/lib/icecube/STTools/seededRT/__init__.py:docstring of icecube.STTools.seededRT.doSeededRTCleaning_RecoPulse_:18: ERROR: Unknown target name: \"seed_with_hit_series_map_hits_from_frame_i3recopulse\".\n}}}", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067158057333", 
    "component": "combo reconstruction", 
    "summary": "[STTools] python names ending with an underscore confuse sphinx", 
    "priority": "normal", 
    "keywords": "documentation", 
    "time": "2016-06-10T07:50:37", 
    "milestone": "", 
    "owner": "mwolf", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160615T18:39:57: olivas changed component from "cmake" to "icerec"

**Comment by icecube-trac** [comment]
Trac update at 20160816T06:43:35: 
* kjmeagher changed status from "new" to "closed"
* kjmeagher commented:

> In changeset 149269:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="149269"
> remove underscores from the end of member functions which was confusing the documentation. fixes #1730
> ```


* kjmeagher changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:38: 
* olivas commented:

> Milestone August 2016 deleted


* olivas changed milestone from "August 2016" to ""
